datastep/bin/make_new_step.py

```diff
@@ -176,32 +176,32 @@ def run(self, **kwargs):
 ###############################################################################
 
 
+def _find_steps_dir():
+    filters = [".git", ".egg", "docs", "localstaging", ".ipynb"]
+
+    for d in Path.cwd().iterdir():
+        if d.is_dir():
+            if not any(f in d.name for f in filters):
+                for subd in d.iterdir():
+                    if subd.name == "steps":
+                        return subd
```

Comment:
i don't love that this short circuits on the first find... what about like `docs/steps` or something?  i think we should find all of them and assert `len(found_steps_dirs_list) == 1` or something